gulp/tasks/release.js

```diff
@@ -1,211 +0,0 @@
-/*
-Copyright 2019 Adobe. All rights reserved.
-This file is licensed to you under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License. You may obtain a copy
-of the License at http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software distributed under
-the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
-OF ANY KIND, either express or implied. See the License for the specific language
-governing permissions and limitations under the License.
-*/
-module.exports = function(gulp) {
-  const argv = require('minimist')(process.argv.slice(2));
-  const bump = require('gulp-bump');
-  const fs = require('fs');
-  const git = require('gulp-git');
-  const inquirer = require('inquirer');
-  const semver = require('semver');
-  const spawn = require('child_process').spawn;
-  const PluginError = require('plugin-error');
-  const CWD = process.cwd();
-
-  function getPackageJson() {
-    return JSON.parse(fs.readFileSync('./package.json', 'utf8'));
-  }
-
-  let releaseVersion;
-
-  gulp.task('perform-version-bump', (done) => {
-    const doBump = () => {
-      gulp.src([`${CWD}/package.json`, `${CWD}/package-lock.json`, `${CWD}/version.json`])
```

Comment:
We still need to update https://github.com/adobe/adobe-client-data-layer/blob/master/version.json